### ISSUE TITLE
JSDoc comment block shown as indentation error

### ISSUE THREAD

**Issue by bpatram** [oderwat/vscode-indent-rainbow#17]
When writing javascript and documenting via JSDoc, normally there is an extra space after the indentation. This extra space is highlighted as red, which is correct, but not very intuitive.

```javascript
class MyClass {
	/**
	 * Creates an instance of MyClass.
	 * 
	 * @memberof MyClass
	 */
	constructor() {}
}
```

Example:
![image](https://cloud.githubusercontent.com/assets/5579960/25899755/0b19464a-355f-11e7-863c-e10d2b821864.png)

Is it possible to add a setting to ignore tab errors for comment blocks or have some heuristics to detect if it is a JSDoc comment block?

**Comment by oderwat** [comment]
One could probably add a setting which ignores lines starting with a `*` (and `/`). Not a high priority problem to me though.

**Comment by hagata** [comment]
Added a fix for this in PR #19

**Comment by oderwat** [comment]
Yes he did, and it is a very nice one!
